### DISCUSSION TITLE
Adding repositories with special chars in their names to a custom scope does not work

### DISCUSSION THREAD

**Discussion by achimnol** [community/community#9756] [Other Feature Feedback, Questions, & Ideas] [answered]
I have many repositories whose names contain dot and hyphens, such as `lablup/backend.ai-manager`, `lablup/backend.ai-agent`, so on.

When I try to add them to a new custom scope, it says they are not valid repository names.
I could add a non-existent repository `lablup/backend` which does not contain dots or hyphens, so it seems that the custom scope dialog is validating the repository names with only alphanumeric characters.

Please allow adding repositories with special chars in their names!

**Comment by colinwm** [comment, marked as answer]
Sorry about that! This should be fixed now.

**Reply by achimnol** [reply]
Confirmed working now!
